src/common/libfluxutil/Makefile.am

```diff
@@ -9,11 +9,41 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/src/include \
 	-I$(top_srcdir)/src/common/libccan \
-	-I$(top_builddir)/src/common/libflux
+	-I$(top_builddir)/src/common/libflux \
+	$(LIBUUID_CFLAGS)
```

Comment:
On moving msg_hash out of librouter - librouter is already included in libflux-internal.la so is it necessary to move it?

Reply:
i guess it's not super necessary.  I moved it out just b/c it is generically useful and it feels weird to include a convenience library header from `librouter/msg_hash.h` instead of `libutil/msg_hash.h`.

Reply:
Prob not worth it IMHO.  It's just one convenience library vs another.  :shrug: